jsgl/src/main/java/roboviz/jsgl/io/ByteUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
I see that you've unpacked the jsgl files... We probably want to use the same clang-format config for these.

Reply:
resolved in 3bf50f6